packages/demos-babylon/src/common/demo-common.css

```diff
@@ -1,8 +1,9 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Serif:ital,wght@1,200;1,500&family=Roboto:wght@400;700&display=swap');
```

Comment:
Q: what is the added string's usage?

Reply:
This change reflects that in addition to loading a Google font called "Roboto" I'm also including another font called "Roboto Serif" and only including the italic version of that font face. (BTW, these strings are automatically provided by the fonts.google.com website.)